src/grisp_onewire.erl

```diff
@@ -55,7 +74,16 @@ transaction(Fun) when is_function(Fun) ->
             erlang:raise(Class, Reason, Stacktrace)
     end.
 
-%% Spec: | S | AD,0 ‖ A ‖ DRST ‖ A ‖ Sr | AD,1 ‖ A | <byte> ‖ A\ | P |
+%% @doc <q>Performs a global reset of device state machine logic. Terminates any
```

Comment:
The `<q>` tag here should be removed.

Reply:
It's a quote from the masters data sheet. It's missing the source. How about:

>  According to the data sheet this `<q>` performs ...

Reply:
I think we should rather rewrite it for our readers then. They shouldn't have to know about the data sheet, so a quote would look out of place to them.